### PR TITLE
Assignments can be shown before they accept submissions.

### PR DESCRIPTION
**Pull request by pwnall**

### CODE REVIEW

**Comment by pwnall** [comment]
@spark008 Please take a look.

File: app/helpers/icons_helper.rb [hunk 1]
```diff
@@ -87,7 +87,15 @@ def help_icon_tag
 
   # Shown on buttons that release some data to the public.
   def release_icon_tag
-    fa_icon :unlock, title: 'Ship it!'
+    fa_icon :'unlock-alt', title: 'Ship it!'
+  end
+
+  def show_icon_tag
+    fa_icon :eye, title: 'Make visible'
+  end
```
Comment (spark008) [inline app/helpers/icons_helper.rb]: Maybe we should just keep one of either `view_icon_tag` or `show_icon_tag` since they look to be pretty similar.

Comment (pwnall) [inline app/helpers/icons_helper.rb]: It seemed to me that they have different intents -- `show` reveals the data to other users, whereas `view` gives you more data. It is true that the helpers use the same icon, and perhaps we can figure that out later? :sweat:

File: app/views/assignments/_deliverable_release.html.erb [hunk 2]
```diff
@@ -1,19 +1,43 @@
...
           data: { confirm: unrelease_confirmation(assignment) } do %>
-      <%= pull_icon_tag %> Pull
+      <%= pull_icon_tag %> Lock submissions
     <% end %>
   <% end %>
-<% else %>
-  <%= form_for assignment, url: release_assignment_path(assignment,
+<% end %>
+
+<% if assignment.scheduled? %>
```
Comment (spark008) [inline app/views/assignments/_deliverable_release.html.erb]: Can we batch the forms that are both conditioned on `released?` in `if/else` blocks? And likewise for `scheduled?`?

**Pull request merged**